modules/core/src/utils/json-loader.ts

```diff
@@ -4,10 +4,9 @@ function isJSON(text: string): boolean {
   return (firstChar === '{' && lastChar === '}') || (firstChar === '[' && lastChar === ']');
 }
 
-// TODO - replace with the version in loaders.gl
+// A light weight version instead of @loaders.gl/json (stream processing etc.)
```

Comment:
I think the loaders.gl core module already has a light weight JSON.parse based JSON loader.

Reply:
The only one I can find is `JSONLoader` which contains `parseInBatches` (can't tell how much bundle size impact it has, I can test)